release/nightly_tests/dataset/multi_node_train_benchmark.py

```diff
@@ -31,6 +40,18 @@ def parse_args():
         type=str,
         help="Input file type; choose from: ['image', 'parquet']",
     )
+    parser.add_argument(
+        "--repeat-ds",
+        default=1,
+        type=int,
+        help="Read the input dataset n times, used to increase the total data size."
+    )
+    parser.add_argument(
+        "--read-task-cpus",
+        default=2,
```

Comment:
Probably make this 1 by default? Or just remove, I'm not sure we need it, although it'd be good to confirm that changing this parameter doesn't affect the final throughput if the output dataset is cached.

Reply:
i intended to use this parameter for the case where we want to test caching with extra CPU nodes, where we can specify the number of CPUs to actually use -- do you think instead of doing that, i should just run it as a separate release test with a different compute config?